app/controllers/bus_stops_controller.rb

```diff
@@ -48,6 +49,7 @@ def name_search
     end
   end
 
+  # TODO: fill in csv method, then implement BSFs
```

Comment:
Confused what this means. What are we waiting on, and what is it preventing us from doing?

Isn't implementing bus stop fields the whole point of the rewrite?

Reply:
I'm not really waiting on anything. I wanted to get the first half of this out there. It isn't finished yet. If you don't want the PR up, I can definitely take it down.

Reply:
Okay, I see. That's fine.